spec/rubocop/cop/lint/void_spec.rb

```diff
@@ -254,4 +254,14 @@ def foo=(rhs)
       nil
     RUBY
   end
+
+  it 'accepts method with range block' do
+    expect_no_offenses(<<~RUBY)
+      def foo
+        1..100.times.each { puts 1 }
+        1...100.times.each { puts 1 }
```

Comment:
Can you split it into irange and erange tests?

Reply:
Done.